package.json

```diff
@@ -120,6 +120,6 @@
     "eslint-import-resolver-typescript": "^2.0.0"
   },
   "engines": {
-    "node": "12.x"
+    "node": ">=12.x"
```

Comment:
Is this change necessary or is it to help for local development?
I am generally against having a "broad" version of node in here. If we think we are ready to update to node 14, then we should aim to set the engine to `14.x` instead and fully test end-to-end that this is not breaking anything (build, deploy, ...)

Reply:
I had issues setting up locally. No problem removing this constraint